src/ganj/HumanRiddleMaker.java

```diff
@@ -0,0 +1,30 @@
+package ganj;
+
+import java.util.Scanner;
+
+public class HumanRiddleMaker implements RiddleMaker {
+    private int secretNumber;
+    @Override
+    public void thinkOfNumberBetween(int from, int to) {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.println("Let's guess a number from " + from + " to " + to);
+            secretNumber = Integer.parseInt(scanner.nextLine());
+            if (secretNumber >= from && secretNumber <= to) {
+                break;
+            } else {
+                System.out.println("Entered number is out of range!");
+            }
+        }
+    }
+
+    @Override
+    public CompareResult check(int number) {
+        if (number == secretNumber) {
```

Comment:
Here you need to print the number to the player and read the input with the result.